examples/volumes/vsphere/README.md

```diff
@@ -353,6 +354,168 @@
       pvpod       1/1     Running   0          48m        
       ```
 
+### Virtual SAN policy support inside Kubernetes
+  __Note: Here you don't need to create vmdk it is created for you.__
```

Comment:
We are not using the same capability names as VSAN policy does. For eg. for stripeWidth we use diskStripes etc. So it might create some confusion is what I felt. 

I think here we need to point to kubernetes.io docs. Anyways, adding it for now, later when the documentation is ready in kubernetes.io we can replace it.